### ISSUE TITLE
Database Errors

### ISSUE THREAD

**Issue by Cpaztik** [AsYetUntitled/Framework#464]
This was tested on my server and on a vanilla copy and I keep getting this error

Not exactly sure, but there is a database error that places players in jail, strips them of all gear and money, and sets it to 5000

### Steps to reproduce the behaviour
1. 
Not sure

`Error in expression <[<null>,<null>,0]>
  Error position: <<null>,<null>,0]>
  Error Invalid number in expression
Error in expression <[<null>,<null>,0]>
  Error position: <<null>,<null>,0]>
  Error Invalid number in expression
Error in expression <playtime_values_request pushBack [_uid, _new];
};
[_uid,_new select 2] call TON_>
  Error position: <_new];
};
[_uid,_new select 2] call TON_>
  Error Undefined variable in expression: _new
File \life_server\Functions\MySQL\fn_queryRequest.sqf [DB_fnc_queryRequest], line 178
Error in expression <) isEqualTo _uid) exitWith {
_x set [1, _time_gathered];
_x set [2, time];
};
} >
  Error position: <_time_gathered];
_x set [2, time];
};
} >
  Error Undefined variable in expression: _time_gathered
File \life_server\Functions\PlayTime\fn_setPlayTime.sqf [TON_fnc_setPlayTime], line 24
Error in expression <fnc_playtime_values_request find [_uid, _new];
if (_index != -1) then {
TON_fnc_>
  Error position: <_new];
if (_index != -1) then {
TON_fnc_>
  Error Undefined variable in expression: _new
File \life_server\Functions\MySQL\fn_queryRequest.sqf [DB_fnc_queryRequest], line 172
"adolf"`

Mission version: 5.0

Game version: 

Branch:

**Comment by Cpaztik** [comment]
> Error in expression <[<null>,<null>,0]>
  Error position: <<null>,<null>,0]>
  Error Invalid number in expression
Error in expression <[<null>,<null>,0]>
  Error position: <<null>,<null>,0]>
  Error Invalid number in expression
Error in expression <playtime_values_request pushBack [_uid, _new];
};
[_uid,_new select 2] call TON_>
  Error position: <_new];
};
[_uid,_new select 2] call TON_>
  Error Undefined variable in expression: _new
File \life_server\Functions\MySQL\fn_queryRequest.sqf [DB_fnc_queryRequest], line 178
Error in expression <) isEqualTo _uid) exitWith {
_x set [1, _time_gathered];
_x set [2, time];
};
} >
  Error position: <_time_gathered];
_x set [2, time];
};
} >
  Error Undefined variable in expression: _time_gathered
File \life_server\Functions\PlayTime\fn_setPlayTime.sqf [TON_fnc_setPlayTime], line 24
Error in expression <fnc_playtime_values_request find [_uid, _new];
if (_index != -1) then {
TON_fnc_>
  Error position: <_new];
if (_index != -1) then {
TON_fnc_>
  Error Undefined variable in expression: _new
File \life_server\Functions\MySQL\fn_queryRequest.sqf [DB_fnc_queryRequest], line 172
"adolf"

**Comment by Alex715** [comment]
i have already got undefined variable for _time_gathered var, random for me but very rare.

check insert_time in ur players table, if there is not a null result instead of a number?

**Comment by Cpaztik** [comment]
My insert_time is currently working, getting numbers like 
2018-01-09 17:09:46
for every player. Am I supposed to be getting null?

**Comment by Cpaztik** [comment]
okay, but im constantly getting them. Like I atleast have 2 players everyday who get the null issue, and it wipes their whole database.

**Comment by BoGuu** [comment]
The latter certainly shouldn't happen. Are you using fresh files with extDB3?

**Comment by Cpaztik** [comment]
I have just replaced my database server, I figured it there was something going on with MySQL community servers, and made a Maria SQL. It seems to be working better for now. will update if the problem comes back. If it does, I plan on replacing my extDB3 with a fresh copy

**Comment by Cpaztik** [comment]
Nope. I also replaced extDB3 with a fresh copy and its still not working. Now its like one out of every 3 players end up with a null value somewhere in their playtime data. Where its supposed to be '''[0,0,0]''' I get '''[0,null,null]'''  '''[0,null,0]''' and sometimes even  '''[0,0]'''.
